command/pr.go

```diff
@@ -367,6 +374,43 @@ func prClose(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func prReopen(cmd *cobra.Command, args []string) error {
+	ctx := contextForCommand(cmd)
+	apiClient, err := apiClientForContext(ctx)
+	if err != nil {
+		return err
+	}
+
+	baseRepo, err := determineBaseRepo(cmd, ctx)
+	if err != nil {
+		return err
+	}
+
+	pr, err := prFromArg(apiClient, baseRepo, args[0])
+	if err != nil {
+		return err
+	}
+
+	if pr.State == "MERGED" {
+		fmt.Fprintf(colorableErr(cmd), "%s Pull request #%d can't be reopened because it was already merged.\n", utils.Red("!"), pr.Number)
+		return nil
```

Comment:
I wasn't sure if this should return an error or not. What do you think @vilmibm @mislav?

Reply:
In general, I think we should fail with a non-zero status every time we are unable to perform an operation, or (more precisely) if we are unable to reach the end-state that the user requested.